pom.xml

```diff
@@ -78,6 +78,16 @@
             <artifactId>commons-lang3</artifactId>
             <version>3.7</version>
         </dependency>
+        <dependency>
+            <groupId>javax.xml.bind</groupId>
+            <artifactId>jaxb-api</artifactId>
+            <version>2.2.11</version>
```

Comment:
Should we use the version of 2.3.x? https://mvnrepository.com/artifact/javax.xml.bind/jaxb-api